omod-common/src/test/java/org/openmrs/module/webservices/rest/web/api/impl/RestServiceImplTest.java

```diff
@@ -205,8 +206,7 @@ public void getResourceByName_shouldReturnResourceForGivenNameAndIgnoreUnannotat
 		resources.add(UnannotatedAnimalResource.class);
 		
 		when(openmrsClassScanner.getClasses(Resource.class, true)).thenReturn(resources);
-		setCurrentOpenmrsVersion("1.9.10");
```

Comment:
What are you replacing setCurrentOpenmrsVersion with?

Reply:
@dkayiwa this is related to https://github.com/openmrs/openmrs-module-webservices.rest/pull/670#issuecomment-3187812725